ytdl_nfo/nfo.py

```diff
@@ -95,4 +99,4 @@ def get_nfo(self):
 
 
 def get_config(extractor):
-   return Nfo(extractor)
+    return Nfo(extractor.replace(":tab", ""))
```

Comment:
I started using [yt-dlp](https://github.com/yt-dlp/yt-dlp) over youtube-dl because the new fork fixes some download throttling issues I was running into.

However, the "extractor" field in yt-dlp `.info.json` files seems to be `youtube:tab` instead of `youtube`. But since the info formats are very nearly identical (except for one important difference, covered in another part of the diff), I don't think there's any reason to use a separate extractor. Hence this code, that routes `youtube:tab` through the normal `youtube` extractor yaml.